src/util/cmdlineargs.cpp

```diff
@@ -40,7 +40,7 @@ CmdlineArgs::CmdlineArgs()
           // TODO(XXX) Trailing slash not needed anymore as we switches from String::append
           // to QDir::filePath elsewhere in the code. This is candidate for removal.
           m_settingsPath(
-                  QStandardPaths::writableLocation(QStandardPaths::DataLocation)
+                  QStandardPaths::writableLocation(QStandardPaths::AppDataLocation)
```

Comment:
Please put this into a separate PR since it probably needs more discussion. This would be a breaking change because AppDataLocation is different from DataLocation (=AppLocalDataLocation):

> DataLocation: Returns the same value as AppLocalDataLocation. This enumeration value is deprecated. Using AppDataLocation is preferable since on Windows, the roaming path is recommended

Reply:
ok i'll break up the PR into 2 separate ones... but note, `QStandardPaths::DataLocation` doesn't exist in qt `6.2.1` so w/o this patch it does not compile :)

Reply:
Thank you!